include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -959,6 +959,34 @@ class CommonOperatorLoader : public ProtobufLoader {
     return llvm::Error::success();
   }
 
+  llvm::Error loadConstantOfShape(const std::string &typeName, const OpType &op,
```

Comment:
Should this be in OnnxModelLoader instead?

Reply:
Caffe2 does have the similar operator ConstantOfShape.